composer.json

```diff
@@ -7,7 +7,7 @@
     "autoload": {
         "psr-0":{
```

Comment:
would it be an idea to switch to psr-4 autoloading now that changes are made here, to make it future proof?